### ISSUE TITLE
NoClassDefFoundError: io/fabric8/maven/util/MavenUtils when provisioning child container 

### ISSUE THREAD

**Issue by rawlingsj** [fabric8io/fabric8#533]
Version: Fabric8 Container (1.0.0.redhat-313) on Openshift

I tried creating a child container and assigning a custom profile which referenced an offline repo on the fabric server containing my artifacts.  Received the following error..

```
Unable to update agent
Stack Trace:
io.fabric8.agent.utils.MultiException: Error while downloading artifacts
  at io.fabric8.agent.utils.AgentUtils$ArtifactDownloader.await(AgentUtils.java:208)[io.fabric8:fabric-agent:1.0.0.redhat-313]
  at io.fabric8.agent.DeploymentBuilder.download(DeploymentBuilder.java:149)[io.fabric8:fabric-agent:1.0.0.redhat-313]
  at io.fabric8.agent.DeploymentAgent.doUpdate(DeploymentAgent.java:474)[io.fabric8:fabric-agent:1.0.0.redhat-313]
  at io.fabric8.agent.DeploymentAgent$2.run(DeploymentAgent.java:240)[io.fabric8:fabric-agent:1.0.0.redhat-313]
  at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)[:1.7.0_45]
  at java.util.concurrent.FutureTask.run(FutureTask.java:262)[:1.7.0_45]
  at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)[:1.7.0_45]
  at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)[:1.7.0_45]
  at java.lang.Thread.run(Thread.java:744)[:1.7.0_45]
Caused by: java.lang.NoClassDefFoundError: io/fabric8/maven/util/MavenUtils
    at io.fabric8.fab.MavenRepositorySystemSession.(MavenRepositorySystemSession.java:61)
    at io.fabric8.fab.MavenResolverImpl.createRepositorSystemSession(MavenResolverImpl.java:435)
```

**Comment by TimothyWatson** [comment]
Guys, I'm not following exactly how you resolved that dependency.  I have integrated fabric8 to my pom and the server to my settings.xml, but running the "fabric8:deploy" doesn't fix the deployed profile in Openshift Online.  Assigning that profile to a container results in the same "Classpath" error.  I don't see a "mavenUtils" (or any "maven") Features to add to the Container.  What am I missing?
By the way, I followed jstachan's Vimeo (thanks!) to work out the deployment details, but I'm still missing something...
